src/index.js

```diff
@@ -58,14 +58,16 @@ function findHandlers(el, selector, event, callback) {
 function removeEvent(el, selector, event, callback) {
   const eventName = parse(event).e;
 
+  if (!el._dtId) return false;
+  const elHandlers = handlers[getDtId(el)];
   const matchedHandlers = findHandlers(el, selector, event, callback);
   matchedHandlers.forEach(handler => {
     if (el.removeEventListener) {
       el.removeEventListener(eventName, handler.delegator || handler.callback);
     } else if (el.detachEvent) {
       el.detachEvent('on' + eventName, handler.delegator || handler.callback);
     }
-    handler = null;
+    elHandlers.splice(elHandlers.findIndex((item) => handler === item), 1);
```

Comment:
`.findIndex()` is only supported in modern browsers, should it be a compatibility concern for us?